server/src/services/duplicate.service.ts

```diff
@@ -15,8 +15,24 @@ import { usePagination } from 'src/utils/pagination';
 export class DuplicateService extends BaseService {
   async getDuplicates(auth: AuthDto): Promise<DuplicateResponseDto[]> {
     const res = await this.assetRepository.getDuplicates({ userIds: [auth.user.id] });
-
-    return mapDuplicateResponse(res.map((a) => mapAsset(a, { auth, withStack: true })));
+    const uniqueAssetIds: string[] = [];
```

Comment:
Naming nitpick: These aren't necessarily _unique_, right? Not sure what a better name would be though.

Reply:
`singletonDuplicateIds`?